fdbt-dev/data/netexData/carnetReturn.xml

```diff
@@ -26,7 +26,6 @@
       <RequestDetail>AllWithXref</RequestDetail>
     </RequestPolicy>
   </PublicationRequest>
-  <Description>Example  of simple point to point fares</Description>
```

Comment:
Did we mean to remove thise?

Reply:
This was an example value from the templates - it's not required and we don't collect a value so removed it